source/glimmer-components/index.md

```diff
@@ -0,0 +1,37 @@
+## HTML elements as an abstraction for text documents
+
```

Comment:
I think this introduction needs to talk about the programming model instead of components as high level abstractions for html.